src/Domain/User/UserWithPermissions.ts

```diff
@@ -0,0 +1,9 @@
+export type UserWithPermissions = {
+  uuid: string,
+  permissions: UserPermissions,
```

Comment:
`Array<UserPermission>` or `UserPermission[]` is fine. No need to introduce collection type aliases

Reply:
Removed.

Originally this was an object type and I left the alias in, so I wouldn't have to update use sites in case I'd want to go back to that.